### PR TITLE
Remove some unused props

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

**Comment by weaverryan** [comment]
Good catch! But, let's remove the argument entirely from the constructor. Then, there are 2 other references to `new EntityRegenerator()` that need to be updated: in `MakeEntity` and `EntityRegeneratorTest`.

**Comment by dunglas** [comment]
Removing the argument is a BC break. Is it allowed?

**Comment by weaverryan** [comment]
Ah, yes. The class is marked as `@internal`. Almost the entirely library is marked as `@internal`. We purposely don't support using the classes of the library to build on top of.

File: src/Maker/MakeEntity.php [hunk 1]
```diff
@@ -42,12 +42,10 @@
 {
     private $fileManager;
     private $doctrineHelper;
-    private $projectDirectory;
 
     public function __construct(FileManager $fileManager, string $projectDirectory, DoctrineHelper $doctrineHelper)
```
Comment (dunglas) [inline src/Maker/MakeEntity.php]: This one cannot be removed because this class isn't `@internal` and  `$projectDirectory` isn't the last argument.

**Comment by weaverryan** [comment]
Thanks @dunglas!

**Pull request merged**